cwel/style/variable/_vbl-colors.scss

```diff
@@ -87,3 +87,9 @@ $c_icon_color_1: $c_primary;
 $c_icon_button_color_1: $white;
 $c_icon_button_color_2: $c_primary;
 
+// Image
+$c_image_bg: $c_bg-light;
+
+// Display
+$c_display_light_bg: $c_bg-light;
```

Comment:
@NathanRodger I would keep variables for a component in the component's style file: `~/cwel/component/display/display.scss`, and add a `!default` flag so it's easier to theme.

Reply:
How are we meant to change the values for other themes if we do it this way?

Reply:
`$c_display_light_bg: $c_bg_light !default;`